products/libllbuild/include/llbuild/db.h

```diff
@@ -0,0 +1,117 @@
+//
+//  db.h
+//  llbuild
+//
+//  Copyright © 2019 Apple Inc. All rights reserved.
```

Comment:
Please follow the attribution style used in the rest of llbuild, like this (for headers, the style varies a tad for C++ headers vs C++ sources):
```
//===- buildsystem.h ----------------------------------------------*- C -*-===//
//
// This source file is part of the Swift.org open source project
//
// Copyright (c) 2015 - 2017 Apple Inc. and the Swift project authors
// Licensed under Apache License v2.0 with Runtime Library Exception
//
// See http://swift.org/LICENSE.txt for license information
// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
//
//===----------------------------------------------------------------------===//
```